Source/objects.cpp

```diff
@@ -1046,66 +1046,46 @@ void AddL4Goodies()
 
 void AddLazStand()
 {
-	int v0;        // edi
-	int v1;        // esi
-	signed int v2; // ebx
-	int v3;        // edx
-	int v4;        // edi
-	signed int v5; // [esp+Ch] [ebp-Ch]
-	int v6;        // [esp+10h] [ebp-8h]
-	signed int v7; // [esp+14h] [ebp-4h]
+	int xp, yp, xx, yy;
+	int cnt;
+	BOOL found;
 
-	v6 = 0;
-	while (1) {
-		v5 = 1;
-		v0 = random(139, 80) + 16;
-		v1 = random(139, 80) + 16;
-		v2 = -3;
-		do {
-			v7 = -2;
-			v3 = v2 + v1;
-			do {
-				if (!RndLocOk(v7 + v0, v3))
-					v5 = 0;
-				++v7;
-			} while (v7 <= 3);
-			++v2;
-		} while (v2 <= 3);
-		if (v5)
+	cnt = 0;
+	while (TRUE) {
+		found = 1;
+		xp = random(139, 80) + 16;
+		yp = random(139, 80) + 16;
+		for (yy = -3; yy <= 3; yy++) {
+			for (xx =-2; xx <= 3; xx++) {

```

Comment:
```suggestion
			for (xx = -2; xx <= 3; xx++) {
```
These stylistic changes can be automatically applied with clang-format.

Reply:
@mewmew have added you as collaborator hopefully. Not really easy to do on mobile unfortunately :)

Reply:
Feel free to ad me also, then we can have 24h coverage 😁